### PR TITLE
Switch from optparse to argparse

### PR DESCRIPTION
**Pull request by richford**
This PR switches from optparse to argparse in afqbrowser-assemble and afqbrowser-run. It also resolves issue #99.

### CODE REVIEW

**Comment by richford** [comment]
I wasn't able to run afqbrowser-assemble on a mat file that @jyeatman sent to us. I resolved that error by switching from opts.source to the only element of args. But then I decided to switch entirely from the optparse module to the argparse module. optparse is deprecated since 2.7. My apologies @arokem for not checking on this on your original PR #95 and #97.

File: README.md [hunk 1]
```diff
@@ -31,5 +31,5 @@ After installing the software, you can run the program on mat files generated by
     afqbrowser-assemble afq.mat
     afqbrowser-run
 
-Open a browser pointing to http://localhost:8080, to view the visualization of
```
Comment (arokem) [inline README.md]: Maybe we should set the default to 8080? 8888 is usually what Jupyer chooses per default, and we might want to set up machines that can do both (e.g., on AWS). 

We'd probably also need to change here: https://github.com/yeatmanlab/AFQ-Browser/blob/master/afqbrowser/browser.py#L147

**Review by arokem — APPROVED** [review]
Great!

**Comment by arokem** [comment]
Travis error seems spurious. I kicked that bot back into action.

File: README.md (hunk 1 above)
Comment (richford) [reply README.md]: Agreed. I'll change the port in both places and push. Thanks for catching that.

**Comment by richford** [comment]
K. Changed the ports. Do you want me to fork pyAFQ and submit a PR for the same switch?

**Comment by arokem** [comment]
That would be awesome.

**Pull request merged**